docs/developer-docs/multi-chain/overview.mdx

```diff
@@ -0,0 +1,96 @@
+
+Chain fusion augments smart contracts and decentralized applications (dapps) on non-ICP chains with the unique capabilities of ICP, including:
+
+- **Web2-like user experience**: Due to low cost, fast finality, and the reverse gas model, interacting with a smart contract on ICP  closely resembles the user experience of traditional web services. By complementing non-ICP smart contracts with user interfaces served from ICP, the user can interact with these smart contracts using a standard browser, without installing a crypto wallet or relying on a single point of trust.
+
+- **Performance**: Smart contracts on the IC possess the power to support a vast range of web services, from social networks to full-fledged AI models. Thus, chain fusion allows to build more powerful dapps combining non-ICP smart contracts with the decentralized compute power of ICP.
+
+- **Storage**: Using a competitive pricing model, a single smart contract on ICP offers up to 400GiB of storage, enabling all application data to be stored on-chain. Through chain fusion, these decentralized storage capacities are made available to non-ICP dapps.
+
+- **Autonomy**: Unlike traditional blockchains, smart contracts on ICP  are capable of scheduling messages for themselves at any given interval. This functionality allows smart contracts to operate without the need for user inputs or arbitrage to maintain their services. Chain fusion provides this autonomy to non-ICP smart contracts without sacrificing decentralization.
+
+- **Interoperability with Web2**: ICP’s HTTP outcall feature permits canisters to reach out to any web service in a replicated fashion, using deterministic response results. Chain fusion brings this Web2-Web3 interoperability to non-ICP dapps. 
+
+The following examples illustrate the variety of use cases enabled by chain fusion:
+
+- **Trustless cronjob service**: Many Ethereum smart contracts depend on periodic activation of certain functions (such as loan liquidations, batching, etc.). These activations usually occur via arbitrageurs or external centralized services like Gelato.
+
+- **A decentralized borrowing protocol**: A DeFi protocol issuing a stablecoin backed by the world-class asset Bitcoin as collateral.
+
+- **Multi-chain asset DEX**: A decentralized exchange providing a uniform user experience, facilitating effortless and efficient trading of assets from different blockchains.
+
+- **In-browser multi-chain wallet**: Similar to a DEX, a single browser-based wallet that manages assets from different and unrelated blockchains.
+
+- **On-chain frontend**: Hosting an immutable or DAO-managed frontend for a smart contract on another blockchain.
+
+- **Decentralized oracle**: Linking Web2 services and smart contracts on other blockchains via HTTP outcalls.
+
+
+## How does it work and why is it secure?
+
+<div class="text--center">
```

Comment:
if you want the text centered, I believe this should be `text-center` with a single dash. https://tailwindcss.com/docs/text-align

Reply:
Good catch, thanks!